test/controllers/notifications_controller_test.rb

```diff
@@ -12,8 +12,8 @@ class NotificationsControllerTest < ActionController::TestCase
   end
 
   test "should assign unapproved sign slides" do
-    users(:two).signs << signs(:one)
-    signs(:one).slides << slides(:two)
+    users(:two).signs << signs(:default)
```

Comment:
@ziegl107 We have to bite the bullet at some point. The longer we wait the less likely it ever gets done. 

The fixtures for `ross` and `james` are not substantively different than `sign_owner` and `non_sign_owner`:

https://github.com/chapmanu/signage/blob/beae5dcb3bbe3b3a53c21f0d7e88cb1371668bdb/test/fixtures/users.yml

Also, do your best to get rid of slide fixture `two`.